libmscore/score.cpp

```diff
@@ -1767,26 +1767,30 @@ void Score::removeAudio()
 
 bool Score::appendScore(Score* score, bool addPageBreak, bool addSectionBreak)
       {
-      if (parts().size() < score->parts().size() || staves().size() < score->staves().size())
+      if (parts().size() < score->parts().size() || staves().size() < score->staves().size()) {
+            qDebug("Score to append has %d parts and %d staves, but this score only has %d parts and %d staves.", score->parts().size(), score->staves().size(), parts().size(), staves().size());
             return false;
-      TieMap tieMap;
+      }
 
-      MeasureBase* lastMeasure = last();
-      if (!lastMeasure)
+      if (!last()) {
```

Comment:
I got rid of the variable "lastMeasure" since I realized that it was poorly named, since is really dealing with MeasureBase objects, not Measures.  Then I realized no need for an actual variable since last() suffices.  So there is no actual semantic difference here.